authors/apps/authentication/tests/test_views.py

```diff
@@ -40,3 +40,25 @@ def test_login_with_invalid_user_fails(self):
         self.assertDictEqual(resp.data, expected_response)
         self.assertIn(
             "A user with this email and password was not found.", str(resp.data))
+
+    # test whether a token is generated on login
+    def test_login_user_login_token(self):
+        user=User.objects.create_user(username='sampleuser', email='user@sprinters.ug',password='butt3rfly1')
+        user.is_active=True
+        user.save()
+
+        response = self.client.post(
+            reverse('auth:login'),
+            data=self.user_data,
+            format='json')
+        self.assertIn('token', response.data)
```

Comment:
@sebbss the variable `token` is not defined anywhere so this test is inevitably fails. Consider changing to `auth_token` that is defined in `serializers.py` and `models.py` for token.
Same goes for `generate_auth_token`

Reply:
i have rebased with develop and made all the tests pass